modules/sfp_countryname.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_countryname
+# Purpose:      SpiderFoot plug-in for scanning retreived content by other
+#               modules (such as sfp_iban, sfp_phone, sfp_whois) and identifying country names
+#
+# Author:      Krishnasis Mandal <krishnasis@hotmail.com>
+#
+# Created:     28/04/2020
+# Copyright:   (c) Steve Micallef
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import re
+import json
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+import phonenumbers
+from phonenumbers.phonenumberutil import region_code_for_country_code
+
+class sfp_countryname(SpiderFootPlugin):
+    """Country Name Extractor:Footprint,Investigate,Passive:Content Analysis::Identify country names in any obtained data."""
+
+    # Default options
+    opts = {
+        # options specific to this module
+        'coHosted' : True,
+        'affiliate' : True,
+        'nonCountryTLD' : True,
+        'nonCountryTLDDefault' : "United States"
+    }
+
+    # Option descriptions
+    optdescs = {
+        'coHosted' : "Include searching of country name from CO_HOSTED_* data sources",
+        'affiliate' : "Include searching of country name from AFFILIATE_* data sources",
+        'nonCountryTLD' : "Parse TLDs not associated with any country as default country domains",
+        'nonCountryTLDDefault' : "Default value for TLDs not associated with any country"
+    }
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+
+        # Clear / reset any other class member variables here
+        # or you risk them persisting between threads.
+
+        # Override datasource for sfp_countryname module
+        self.__dataSource__ = "Target Website"
```

Comment:
Probably get rid of this since you have the logic down below to override it on a case-by-case basis. There is no sane default for this module.